### ISSUE TITLE
macOS 13.3 no longer includes the /private/etc/sudoers.d path

### ISSUE THREAD

**Issue by Nino-K** [rancher-sandbox/rancher-desktop#4377]
Some users reported facing the following issue with macOS Ventura during the installation.

```
Error Starting Kubernetes
Error: Command failed: /bin/sh -xec 'cp "/var/folders/2k/cdjg6vvj0glb1zzz7n0x15d80000gn/T/rd-sudoers19106489.txt" /private/etc/sudoers.d/zzzzz-rancher-desktop-lima && rm -f "/var/folders/2k/cdjg6vvj0glb1zzz7n0x15d80000gn/T/rd-sudoers19106489.txt"'
+ cp /var/folders/2k/cdjg6vvj0glb1zzz7n0x15d80000gn/T/rd-sudoers19106489.txt /private/etc/sudoers.d/zzzzz-rancher-desktop-lima
cp: /private/etc/sudoers.d/zzzzz-rancher-desktop-lima: No such file or directory
```
This is due to the path `/private/etc/sudoers.d` no longer existing on Ventura. As a result, we should either fix or document this so that our users are aware that they would need to manually create that path.

**Comment by cblacknygc** [comment]
Dir appears on macOS 12.6.3 and 13.2.1. I believe the change is new to macOS 13.3, not all of Ventura 13.x. Fix to work on all versions would be checking for dir first and only creating if it doesn't exist.

**Comment by jandubois** [comment]
Already fixed in `main`. Duplicate of #4316.

**Comment by jasikpark** [comment]
I'm on macOS 13.3 using `rancher@1.8.1` installed via `brew install rancher` and still seeing this bug behavior:

```
Error: Command failed: /bin/sh -xec 'cp "/var/folders/6m/bb4xw4d52dv97pcs4269jdfc0000gn/T/rd-sudoers83311439.txt" /private/etc/sudoers.d/zzzzz-rancher-desktop-lima && rm -f "/var/folders/6m/bb4xw4d52dv97pcs4269jdfc0000gn/T/rd-sudoers83311439.txt"'
+ cp /var/folders/6m/bb4xw4d52dv97pcs4269jdfc0000gn/T/rd-sudoers83311439.txt /private/etc/sudoers.d/zzzzz-rancher-desktop-lima
cp: /private/etc/sudoers.d/zzzzz-rancher-desktop-lima: No such file or directory
```

should I install a more up-to-date version to get that fix / just create the directory myself?

**Comment by jandubois** [comment]
> should I install a more up-to-date version to get that fix / just create the directory myself?

You can install the `1.9-tech-preview` release, but just creating the directory yourself is probably the simplest fix.

**Comment by jasikpark** [comment]
yeah, creating it myself worked, is this not warranting a patch release? ig anyone on ventura can just find this issue themselves to fix it?

thx for the help 🙏

**Comment by jandubois** [comment]
> is this not warranting a patch release

As I wrote above, the fix is included in https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.9.0-tech-preview if somebody doesn't want to create the directory themselves. Otherwise it will be available in the upcoming 1.9 release.
